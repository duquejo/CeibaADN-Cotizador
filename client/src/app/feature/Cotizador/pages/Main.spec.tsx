import * as React from 'react';
import AdminMainPage from 'app/feature/Admin/pages/Main';
import { ShallowWrapper, shallow } from 'enzyme';

describe('Pruebas sobre <AdminMainPage />', () => {

    let componentWrapper: ShallowWrapper;

    const defaultProps: any = {
        foo: true,
        bar: false,
        history: {
          replace: jest.fn(),
        },
    };

    it('DeberÃ­a renderizar bien el elemento padre', () => {
        componentWrapper = shallow( <AdminMainPage {...defaultProps} /> );
        expect( componentWrapper ).toMatchSnapshot();
    });
});
