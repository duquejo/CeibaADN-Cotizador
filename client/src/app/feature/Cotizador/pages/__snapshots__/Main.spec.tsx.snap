// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre <AdminMainPage /> Debería renderizar bien el elemento padre 1`] = `
<Layout
  description="Administración"
  title="Administrador"
>
  <h1>
    Administrador
  </h1>
  <p>
    En la siguiente interfaz podrá tomar acciones relacionadas con la gestión de 
    <b>
      Calendarios de temporadas altas
    </b>
    ,
    <b>
      Categorías de usuarios
    </b>
     y 
    <b>
      Centros Vacacionales
    </b>
    . 
    <br />
    Los calendarios personalizados gestionados en el módulo de 
    <b>
      Gestión de calendarios
    </b>
     tienen un impacto sobre el cálculo de la cotización final según el 
    <b>
      Centro Vacacional
    </b>
     que lo tenga asignado, así como la 
    <b>
      Categoría de usuarios
    </b>
     relacionada, que le permite realizar a un usuario diferentes cotizaciones según su rango salarial.
  </p>
  <styled.hr />
  <Connect(GestionCalendarios) />
  <Connect(GestionCategoriasUsuarios) />
  <Connect(GestionCentrosVacacionales) />
  <GestionCotizaciones />
</Layout>
`;
